### PR TITLE
ceph-ansible: add add_rgws scenario to CI

### PR DESCRIPTION
**Pull request by rishabh-d-dave**
Add add_rgws scenario to CI so that it can be tested against master.
    
Signed-off-by: Rishabh Dave <ridave@redhat.com>

### CODE REVIEW

**Comment by rishabh-d-dave** [comment]
I created this PR by mistake, I was supposed to create one for RBD mirrors.

This PR will be eventually needed when I add a tox scenario for addding a new RGW node. Till then I am blocking this PR.

**Comment by rishabh-d-dave** [comment]
`dev-centos-non_container-add_rgws` and `dev-centos-container-add_rgws` worked fine locally.

@guits Please review and merge this PR.

**Comment by rishabh-d-dave** [comment]
Assuming that we'll be backporting this PR too, I've added jobs definitions (if that's what it is called) for nautilus as well.

File: ceph-ansible-pipeline/config/definitions/ceph-ansible-pipeline.yml [hunk 1]
```diff
@@ -314,6 +314,48 @@
...
+      - conditional-step:
+          condition-kind: shell
+          condition-command: |
+            #!/bin/bash
+            set -x
+            # if the target branch is master we RUN these tests.
+            if [[ "$ghprbTargetBranch" =~ "stable-" ]]; then
+              exit 1
+            fi
+            git diff --name-only $(git show HEAD | grep Merge | head -n 1 | cut -d ':' -f2) | grep -E 'infrastructure-playbooks/add-mon|tests/functional/add-rgws'
```
Comment (guits) [inline ceph-ansible-pipeline/config/definitions/ceph-ansible-pipeline.yml]:
```suggestion
            git diff --name-only $(git show HEAD | grep Merge | head -n 1 | cut -d ':' -f2) | grep -E 'tests/functional/add-rgws'
```

File: ceph-ansible-pipeline/config/definitions/ceph-ansible-pipeline.yml [hunk 2]
```diff
@@ -314,6 +314,48 @@
...
+      - conditional-step:
+          condition-kind: shell
+          condition-command: |
+            #!/bin/bash
+            set -x
+            # if the target branch is stable-4.0 we RUN these tests.
+            if [[ "$ghprbTargetBranch" =~ stable-3.[0-2]|master ]]; then
+              exit 1
+            fi
+            git diff --name-only $(git show HEAD | grep Merge | head -n 1 | cut -d ':' -f2) | grep -E 'infrastructure-playbooks/add-rgw|tests/functional/centos/7/add-rgws'
```
Comment (guits) [inline ceph-ansible-pipeline/config/definitions/ceph-ansible-pipeline.yml]:
```suggestion
            git diff --name-only $(git show HEAD | grep Merge | head -n 1 | cut -d ':' -f2) | grep -E 'tests/functional/add-rgws'
```

**Pull request merged**